cal/.vscode/launch.json

```diff
@@ -0,0 +1,44 @@
+// AUTOMATICALLY GENERATED FILE. PLEASE DO NOT MODIFY IT MANUALLY
```

Comment:
git ignore vscode directory

Reply:
I thought we were leaving it because all it is is the extension prompt